WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/ShimmerModifier.swift

```diff
@@ -0,0 +1,90 @@
+import SwiftUI
+
+/// A view modifier that applies an animated "shimmer" to any view,
+/// ideal to be used with the `redacted` modifier.
+/// Author: https://github.com/markiv/SwiftUI-Shimmer
```

Comment:
Does this mean that we would need to include their license too?

Reply:
Probably you are right. Also if it's a [MIT License](https://github.com/markiv/SwiftUI-Shimmer/blob/main/LICENSE) (the most permissive) we should also include the original license. Given that, probably it's better to include this small piece of code using SPM? What do you think?

We can still include the license also doing a copy/paste of the license on top of the file.

Reply:
Including it as an SPM would probably be best. As long as we don't break our builds again. 😄 A separate SPM also informs us that “this is third-party code and we shouldn't touch it”. 

Totally cool if you'd like to do this later and leave it as is for now.

Reply:
I'll handle it in a different PR ;)